### DISCUSSION TITLE
How to create threads from a Message

### DISCUSSION THREAD

**Discussion by tbezierslafosse** [kordlib/kord#919] [Help] [answered]
Hey,

I'm trying to create a thread from a message that my bot sends, but I don't know how, and I'm unable to find any tips so far :(
I have something like :

`      
val message = interaction.channel.createMessage {
    content = "A message"
}
`

And I want to start a Thread starting from here. Any idea how ? :)
Thanks a lot

**Comment by DRSchlaubi** [comment, marked as answer]
You can use [this](https://dokka.kord.dev/core/dev.kord.core.behavior.channel/-text-channel-behavior/start-public-thread-with-message.html?query=open%20suspend%20fun%20startPublicThreadWithMessage(messageId:%20Snowflake,%20name:%20String,%20builder:%20StartThreadWithMessageBuilder.()%20-%3E%20Unit%20=%20{}):%20TextChannelThread) function
